src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/PaymentDropoutController.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.companieshouse.web.accounts.controller.smallfull;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.servlet.view.UrlBasedViewResolver;
+import uk.gov.companieshouse.web.accounts.annotation.PreviousController;
+import uk.gov.companieshouse.web.accounts.controller.BaseController;
+
+import javax.servlet.http.HttpServletRequest;
+
+@Controller
+@PreviousController(ApprovalController.class)
+@RequestMapping("/company/{companyNumber}/transaction/{transactionId}/company-accounts/{companyAccountsId}/small-full/approved-accounts")
+public class PaymentDropoutController extends BaseController {
+
+    private static final String YOUR_FILINGS_PATH = "/user/transactions";
```

Comment:
This line is being flagged as a code smell by Sonar.
http://code-analysis.platform.aws.chdev.org:9000/project/issues?id=uk.gov.companieshouse%3Acompany-accounts.web.ch.gov.uk&issues=AXVHR13obKZCo3W_a8V7&open=AXVHR13obKZCo3W_a8V7&pullRequest=578
"Refactor your code to get this URI from a customizable parameter"
We should either fix this or ignore it.
What do people think?

Reply:
We agreed to leave this as-is, raise a JIRA to decide what to do generally and then apply that across the board.